templates/jmxtrans.service.epp

```diff
@@ -5,10 +5,18 @@ After=syslog.target network.target
 [Service]
 Type=forking
 User=<%= $user %>
+<% if $jmxtrans::systemd_environment_vars =~ Array[String] { -%>
```

Comment:
This should be replaced with an `EnvironmentFile` entry, so people can just use, e.g., `/etc/default/jmxtrans` on Debian or `/etc/sysconfig/jmxtrans` on EL. Then, rather than passing in a list of environment variables and values, they can just pass in a file name.